Sources/Jetworking/Client/Executor/DownloadExecutor/Background/BackgroundDownloadExecutor.swift

```diff
@@ -0,0 +1,87 @@
+import Foundation
+
+final class BackgroundDownloadExecutor: NSObject, DownloadExecutor {
```

Comment:
Maybe we could add a documentation when we should use a Background Downloader and what's so special about downloading files in background.

Reply:
Added a first documentation which might be enhanced when finally using it.